src/assets/scripts/client/airport/AirportWind.js

```diff
@@ -0,0 +1,35 @@
+import AirportModel from './AirportModel';
+import { calculateNextWind } from '../utilities/windUtilities';
+
+/**
+ * Mutable array that holds the information on the airport's current wind.
+ */
+export let airportWind = {
+   speed: 10,
+   angle: 0
+};
+
+export default class AirportWind {
+    /**
+     * Constructor
+     */
+    constructor() {}
+
+    /**
+     * Sets the initial airportWind, instead of the default.
+     * Also can be used to force-set the wind.
+     * 
+     * @method setStaticWind
+     * @param nextWind {Array} [speed, angle]
+     */
+    setStaticWind(nextWind) {
+        airportWind = nextWind;
+    }
+
+    /**
+     * Updates the wind, using a bell curve.
+     */
+    updateWind(wind) {
+        return calculateNextWind(wind);
```

Comment:
because we now have a class that holds all of our wind logic and values, we can probably bring this `calculateNextWind` function in and use it as a class method.